CHANGELOG.md

```diff
@@ -7,16 +7,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 # Unreleased
 
 
-## Added
+## Added
+
 - Prism is now able to validate the security specification of the loaded document #484
 
 ## Fixed
 
 - Prism is not crashing anymore when referencing the same model multiple times in the specification document #552
+- Prism won't crash anymore in case you're trying to return a primitive type as a `text/plain` #560
```

Comment:
Let's save the term "crash" for the server falling over and throwing awkward exceptions. 
```suggestion
- Prism won't return 406 when users request a simple `text/plain` #560
```